platformio.ini

```diff
@@ -319,3 +319,26 @@ build_flags =
   -D RAPI_PORT=Serial1
   -D RX1=25
   -D TX1=27
+
+[env:wt32-eth01]
```

Comment:
Changes look ok, but can you add to the GitHub workflow: <https://github.com/OpenEVSE/openevse_esp32_firmware/blob/master/.github/workflows/build.yaml#L57>

Reply:
Done!